actions/testmo-run-submit-thread/action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   results:
     description: "directory of JUnit '*.xml' formatted result files"
     required: true
+  thread_resources:
+    description: "XML file with TestMo resources to attach to this thread"
```

Comment:
nit: It's JSON, not XML, unless something has drastically changed in the newer Testmo versions (though it certainly still accepts JSON as other jobs in other projects pass JSON).